pkg/kubelet/im/image_gc.go

```diff
@@ -104,38 +97,57 @@ func newImageManager(runtime container.Runtime, cadvisorInterface cadvisor.Inter
 	if policy.LowThresholdPercent > policy.HighThresholdPercent {
 		return nil, fmt.Errorf("LowThresholdPercent %d can not be higher than HighThresholdPercent %d", policy.LowThresholdPercent, policy.HighThresholdPercent)
 	}
-	im := &realImageManager{
+
+	im := &realImageGC{
 		runtime:      runtime,
 		policy:       policy,
+		minAge:       defaultGCAge,
 		imageRecords: make(map[string]*imageRecord),
 		cadvisor:     cadvisorInterface,
 		recorder:     recorder,
 		nodeRef:      nodeRef,
-		initialized:  false,
 	}
 
 	return im, nil
 }
 
-func (im *realImageManager) Start() error {
-	go util.Until(func() {
-		// Initial detection make detected time "unknown" in the past.
-		var ts time.Time
-		if im.initialized {
-			ts = time.Now()
-		}
-		err := im.detectImages(ts)
```

Comment:
Why are you removing this logic? This part is required to refresh kubelet's internal representation of images up to date.